test/benchmarks/benchmark.rake

```diff
@@ -393,11 +393,15 @@ namespace :benchmarks do
   end
 
   def run_big
-    1000.times { require_unique_file }
+    4000.times { |index| require_unique_file('dog.rb.erb', dog_number: index) }
     # warmup
     3.times { Coverband.report_coverage }
+    dogs = 400.times.map { |index| Object.const_get("Dog#{index}") }
     MemoryProfiler.report do
-      10.times { Coverband.report_coverage }
+      10.times do
+        dogs.each(&:bark)
```

Comment:
@danmayer Added generation of coverage data through executing code from different versions of the Dog class. This feels more realistic as a subset of the code is being executed before each call to report_coverage. Think this would be something useful in master.

Reply:
yeah I think it is great that we continue to push our memory tests. I really need to do a blog post on some of the memory testing and benchmarks we have been working on.